andela/wg-wits
# #159236200 Add drop set support.

#### What does this PR do?
- Add drop set support.
#### Description of Task to be completed?
- Modify UI to enable users to create drop sets.
#### How should this be manually tested?
-  Clone this [branch](https://github.com/andela/wg-wits/tree/ft-add-drop-sets-159236200) and follow the instructions stated in [README](https://github.com/andela/wg-wits/tree/ft-add-drop-sets-159236200) to set up the application.
- Run migrations using `python manage.py migrate`.
- After setting up the application, start the server by running the command `python manage.py runserver` in the terminal with your virtual environment set.
- In the browser, log in and try creating a new exercise in the workout. You should be able to change the exercise type to either `Normal` or `Drop Set` as shown in the screenshots section.

#### What are the relevant pivotal tracker stories?
[159236200](https://www.pivotaltracker.com/story/show/159236200)

#### Screenshots
Before the implementation
<img width="747" alt="screen shot 2018-08-23 at 14 00 53" src="https://user-images.githubusercontent.com/39955305/44521835-2b587200-a6dd-11e8-985f-738f68407f0b.png">

After the implementation
<img width="822" alt="screen shot 2018-08-23 at 13 56 12" src="https://user-images.githubusercontent.com/39955305/44521852-41fec900-a6dd-11e8-809f-d631e304361d.png">

<img width="1010" alt="screen shot 2018-08-23 at 14 00 03" src="https://user-images.githubusercontent.com/39955305/44521864-4cb95e00-a6dd-11e8-8734-82eccd122ecf.png">
